airflow/cli/cli_config.py

```diff
@@ -277,7 +277,12 @@ def string_lower_type(val):
 ARG_NO_BACKFILL = Arg(
     ("--no-backfill",), help="filter all the backfill dagruns given the dag id", action="store_true"
 )
-ARG_STATE = Arg(("--state",), help="Only list the dag runs corresponding to the state")
+ARG_STATE = Arg(
+    ("--state",),
+    help="Only list the dag runs corresponding to the state",
+    metavar="(table, json, yaml, plain)",
+    choices=("table", "json", "yaml", "plain"),
+)
```

Comment:
The choices seem wrong…?

Reply:
you just copied the choices from `ARG_OUTPUT`.
The choice for the dag run state are `running`, `queued`,  `success` and `failed`

Reply:
IMO it's better to add them dynamically:
```python
>>> from airflow.utils.state import DagRunState
>>> tuple(state.value for state in DagRunState)
('queued', 'running', 'success', 'failed')
>>> str(tuple(state.value for state in DagRunState))
"('queued', 'running', 'success', 'failed')"
```
@uranusjr wdyt?

Reply:
plz check now